### DISCUSSION TITLE
Failure to login to your email account!

### DISCUSSION THREAD

**Discussion by realbadfox** [berrystephenw/OctoText#83] [General]
Can't send the testmail, log attached
[octoprint.log](https://github.com/berrystephenw/OctoText/files/6527684/octoprint.log)

**Comment by realbadfox** [comment]
I created a new mail account with outlook.com 
Typed all SMTP data and login informations, but did not work.
![Screen](https://user-images.githubusercontent.com/73101501/119251784-a6aa2380-bba8-11eb-917b-521d484751be.jpg)

**Reply by berrystephenw** [reply]
It's having trouble contacting the SMTP server. It would be helpful to have the whole log file to see what is going on.

**Reply by berrystephenw** [reply]
wait, that message is after trying to login to outlook with your user name and password. You might want to double check your password. Another possibility is that you have to setup an app password. There is a post in the FAQ that describes how to do that.

**Reply by realbadfox** [reply]
i generatet a app password, and what I have to do now with it?

And my password in den Setup is 100% correct!

**Reply by berrystephenw** [reply]
you use your app password as your email password.

**Reply by realbadfox** [reply]
did it, same error. log attached
[octoprint.log](https://github.com/berrystephenw/OctoText/files/6528309/octoprint.log)

**Reply by berrystephenw** [reply]
sorry I forgot to tell you that you have to enable debug messages. Go to to settings->logging and then find OctoText in the bottom dropdowns and add it to the logging. Please do a reboot and send me the new log. The errors in the log you attached are all the same - authentication errors trying to login to the mail server. You might want to take a look on outlook.com, open settings and then sync settings and see if it looks like mine (second attachment). You need to match the SMTP settings.

![image](https://user-images.githubusercontent.com/44931130/119266342-6ce11900-bbb8-11eb-8d1b-32195f36adcb.png)
![image](https://user-images.githubusercontent.com/44931130/119266555-3eb00900-bbb9-11eb-8811-b1a240c300d0.png)

**Comment by realbadfox** [comment]
I hope now the logging is correct.
[octoprint (1).log](https://github.com/berrystephenw/OctoText/files/6528650/octoprint.1.log)

**Comment by realbadfox** [comment]
mail settings are the same on outlook as on your screenshot.

**Reply by berrystephenw** [reply]
There is nothing crazy in the log. Are you using the test button in the settings page or the icon on the navigation bar?

You have to save your settings before you try either one of them or any changes won't take.

**Comment by realbadfox** [comment]
yes, I tried with the app Password, with the original password!
and I tried with the test button in settings page and icon on nav bar!
![settings](https://user-images.githubusercontent.com/73101501/119273033-52835b80-bc09-11eb-9597-057843d46004.jpg)

are my settings correct? see attachemend

**Comment by realbadfox** [comment]
OMG!!! I find the failure, not outlook.com....it is outlook.de
now it is working!

**Reply by berrystephenw** [reply]
Wow - I'm glad you found it, and that is good to know!
